merlin/examples/generator.py

```diff
@@ -42,22 +43,20 @@
 from merlin.examples import examples
 
 
-LOG = logging.getLogger("merlin-example")
+LOG = logging.getLogger(__name__)
 
 EXAMPLE_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), "workflows")
 
 
-def gather_examples():
+def gather_example_dirs():
     result = {}
     for d in os.listdir(EXAMPLE_DIR):
         result[d] = d
     return result
 
 
-def find_example(name):
-    for example in examples.EXAMPLES:
-        if example["name"] == name:
-            return example  # Return only the first example for now.
+def gather_all_examples():
+    return glob.glob(os.path.join(EXAMPLE_DIR, "") + "*/*.yaml")
```

Comment:
any reason this can't be simplified to glob.glob(os.path.join(EXAMPLE_DIR, "*.yaml")) ?

Reply:
Yeah that didn't work, though I found a compromise.